tools/Google.Cloud.Tools.TrimCoverageReport/Program.cs

```diff
@@ -0,0 +1,124 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Xml.Linq;
+
+namespace TrimCoverageReport
+{
+    /// <summary>
+    /// Application to trim the detailed XML coverage report produced by dotCover to remove generated code.
+    /// </summary>
+    class Program
+    {
+        // Attributes used multiple times in the XML
+        const string TotalStatements = "TotalStatements";
+        const string CoveredStatements = "CoveredStatements";
+        const string CoveragePercent = "CoveragePercent";
+
+        static int Main(string[] args)
+        {
+            if (args.Length != 2)
+            {
+                Console.WriteLine("Arguments: <input file> <output file>");
+                return 1;
+            }
+            XDocument doc = XDocument.Load(args[0]);
+            ProcessDocument(doc);
+            // Don't use File.CreateText as that omits the byte order mark.
+            // While byte order marks are nasty, Visual Studio will add it back any time a project file is
+            // manually edited, so it's best if we follow suit.
+            using (var writer = File.CreateText(args[1]))
```

Comment:
I'm confused. The comment above says not to use `File.CreateText`. Should this be something else?